src/main/java/cz/muni/fi/pa165/bluebat/dao/TripDao.java

```diff
@@ -0,0 +1,40 @@
+package cz.muni.fi.pa165.bluebat.dao;
+
+import cz.muni.fi.pa165.bluebat.entity.Trip;
+
+/**
+ * @author Ondrej Vaca
+ * An interface for manipulation with the Trip database table.
+ */
+
+public interface TripDao {
+
+    /**
+     * Creates a new trip record in the database.
+     *
+     * @param trip Trip trip record to be created
+     */
+    void create(Trip trip);
+
+    /**
+     * Updates a trip record in the database
+     *
+     * @param trip Trip trip record to be updated
+     */
+    void update(Trip trip);
+
+    /**
+     * Remove a trip record from the database
+     *
+     * @param trip Trip trip record to be removed
+     */
+    void remove(Trip trip);
```

Comment:
Tady navrhuji jen kosmetickou úpravu přejmenovat remove na delete, ať to máme sjednocené mezi entitami